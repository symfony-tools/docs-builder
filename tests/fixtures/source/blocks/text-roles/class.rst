
:class:`Symfony\\Component\\HttpKernel\\DependencyInjection\\ContainerAwareHttpKernel`
