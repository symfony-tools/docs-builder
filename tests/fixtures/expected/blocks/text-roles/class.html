<p><a href="https://github.com/symfony/symfony/blob/4.0/src/Symfony/Component/HttpKernel/DependencyInjection/ContainerAwareHttpKernel.php" class="reference external" title="Symfony\Component\HttpKernel\DependencyInjection\ContainerAwareHttpKernel" rel="external noopener noreferrer" target="_blank">ContainerAwareHttpKernel</a></p>
